incuna-sass/helpers/_box-sizing.sass

```diff
@@ -1,8 +1,14 @@
 %border-box
-    @include box-sizing(border-box)
+    -webkit-box-sizing: border-box
+    -moz-box-sizing: border-box
+    box-sizing: border-box
```

Comment:
I'm not sure we need helpers such as `box-sizing`, `padding-box`, `content-box` -they are widely supported